src/components/Community/Events/index.js

```diff
@@ -63,9 +59,9 @@ function CommunityEvent({
 
         <Link
           color={theme.color}
+          as={LinkSC}
           href={url}
           target="_blank"
-          rel="noreferrer noopener"
```

Comment:
why remove it here? (and we added it in some other places?)

Reply:
It will be added automatically in Link component for all not relative links
https://github.com/iterative/dvc.org/blob/db97402599db07da2c9421ce094658944a1d0bfd/src/components/Link/index.tsx#L22-L24
It much more safety, than setting it manually every time.

Reply:
got it!